main/src/library/Fixpoint/Interpreter.flix

```diff
@@ -29,6 +29,27 @@ namespace Fixpoint {
         db
 
     def evalStmt(db: Database[v], stmt: RamStmt[v]): Unit & Impure with Order[v], Eq[v], ToString[v] = match stmt {
+        case RamStmt.Facts(fs) =>
+            use Fixpoint/Ast.HeadPredicate.HeadAtom;
+            use Fixpoint/Ast.Constraint.Constraint;
+            use Fixpoint/Ast.PredSym.PredSym;
+            Array.foreach(match Constraint(HeadAtom(PredSym(headSym), denotation, terms), _) -> {
+                let ramSym = RamSym.Full(headSym, denotation);
+                let rel = MutMap.getOrElsePut!(ramSym, MutMap.empty(), db);
+                match denotation {
+                    case Denotation.Relational =>
+                        let tuple = Array.map(headTermValue, terms);
```

Comment:
Whats the story with facts that contain function applications, e.g. `A(f(123)`. Does that work? Do we have tests?

Reply:
I'm not sure if it's covered by any test. I'll write one. But it should be possible to evaluate funapps in facts before Datalog evaluation.

Reply:
Yeah it works just fine :)